### DISCUSSION TITLE
VideoPlayer Component does not work when passing a local path to src

### DISCUSSION THREAD

**Discussion by itsAmjad** [vuepress-theme-hope/vuepress-theme-hope#3053] [Q&A] [answered]
### Checklist

- [X] I am using **LTS** version of Node.js.
- [X] I am using the **latest v2 version** of VuePress and VuePress official plugins.
- [X] I am using the **latest v2 version** of vuepress-theme-hope and all plugins in this repo.
- [X] I am following the docs and I am sure I am using the correct configuration.
- [X] I have searched [FAQ](https://theme-hope.vuejs.press/faq/)

### Package name

vuepress-plugin-components

### Which operating system are you using?

- [ ] macOS
- [X] Windows
- [ ] Linux

### Describe the bug

**Bug Description:** after a successful install of the plugin "vuepress-plugin-components" and adding it to my config.ts I ran into an issue where I can see the video player yet the video was not playing. I am not sure if this component is fully configured to play local videos such as mp4's. I also tired to pass props such as type="video/mp4"

**Steps to re-produce:**
- install the vuepress dependency and  configure it in your config
- have a local video in the same directory as your markdown
- use `<VideoPlayer` `src="yourvideoname.mp4" />` and include it inside your markdown

sorry if any of my description is not clear

**Comment by Mister-Hope** [comment, marked as answer]
If you configure your bundler to make the following work:

```md
<video src="./example.mp4" />
```

Then my component should work the same as well.

**Comment by itsAmjad** [comment]
following that format did not fix the problem unfortunately, looking at the console messages it's giving me "failed to load resource" error. can anyone please try to import a local video and see if it works

**Comment by Mister-Hope** [comment]
You should place your video somewhere and use full link, or place it in public folder and reference with pathname (we will automatically append base for you)

**Comment by itsAmjad** [comment]
This worked perfectly for me, thanks mr.hope.
